yarn.lock

```diff
@@ -1373,15 +1373,19 @@
     react-shadow "^17.1.3"
     rimraf "^3.0.0"
 
-"@geops/geops-ui@0.1.8":
-  version "0.1.8"
-  resolved "https://registry.yarnpkg.com/@geops/geops-ui/-/geops-ui-0.1.8.tgz#4df8f4bb8916d762e8b9667cf744fa8961c3598d"
-  integrity sha512-ms3zDcXThMEoVP4wswdMZNajIlwr1mBlFDmtoe4qS1eN7b6iBgd3I2jJ1zSRSbuhaKMmyw7KX+iJcLv3/jY0zg==
+"@geops/geops-ui@0.1.4":
+  version "0.1.4"
```

Comment:
You have to pin to the last version we used . otherwise we regress.

Reply:
I think, I have all regression cases fixed